README.md

```diff
@@ -1,4 +1,4 @@
-# Dynomite-manager
+# Dynomite Manager
```

Comment:
IMHO, a space reads better.

The repo is obviously `dynomite-manager`. 

However, I thought the product was `Dynomite Manager` (i.e. a "Manager" for Dynomite). For example, its DSE OpsCenter (DataStax) or Cloudera Manager. Most companies use a space with capitalization for each word.

I think the following capitalization and spacing makes for a good product family:

- Dynomite
- Dynomite Manager
- Dyno